### PR TITLE
Leave team modal

### PR DESCRIPTION
**Pull request by TheInvincibleRalph**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/28502

```release-note
NONE
```

### CODE REVIEW

File: server/public/model/config.go [hunk 1]
```diff
@@ -3402,8 +3402,9 @@ func (s *MessageExportSettings) SetDefaults() {
 }
 
 type DisplaySettings struct {
-	CustomURLSchemes []string `access:"site_posts"`
-	MaxMarkdownNodes *int     `access:"site_posts"`
+	CustomURLSchemes        []string `access:"site_posts"`
```
Comment (M-ZubairAhmed) [inline server/public/model/config.go:3405]: I guess you moved two prs into one ? This is not part of this pr

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by TheInvincibleRalph** [comment]
To avoid confusion, @M-ZubairAhmed. I'll close this pull request and create another one.
